src/auth/mas.py

```diff
@@ -116,7 +139,7 @@ def register_user(
         process = container.exec(command=command, working_dir=MAS_WORKING_DIR)
         process.wait_output()
     except ops.pebble.ExecError as exc:
-        logger.error("Error registering new user: %s", exc.stderr)
+        logger.exception("Error registering new user.")
         raise MASRegisterUserFailedError("Error validating MAS configuration.") from exc
```

Comment:
Same here regarding expcetion description and logger.exception

Reply:
Updated!